dist/index.html

```diff
@@ -12,29 +12,40 @@ <h1>Overlook</h1>
     <nav>
       <button>Book a room!</button>
       <button>Check your bookings!</button>
-      <button>Login</button>
     </nav>
   </header>
   <main>
-    <aside>
+    <section class="login">
+      <p>Login</p>
       <form>
-        <p>Filter</p>
-        <input type="date" name="Input date" />
-        <label for="room-type">Room Type
-          <select name="room-type" id="room-type">
-            <option value="none">None</option>
-            <option value="one-bed">One Bed</option>
-          </select>
-        </label>
-        <input type="submit" />
+        <label>Username</label>
+        <input type="text" required />
+        <label for="">Password</label>
+        <input type="password" required>
+        <input type="submit" value="Login">
       </form>
-    </aside>
-    <section class="available-rooms">
-      <h2>Available Rooms</h2>
     </section>
-    <section class="bookings">
-      <h2>Your bookings</h2>
-      <p>Log in to see your bookings!</p>
+    <section class="dashboard hidden">
+      <aside>
+        <form>
+          <p>Filter</p>
+          <input type="date" name="Input date" />
+          <label for="room-type">Room Type
+            <select name="room-type" id="room-type">
+              <option value="none">None</option>
+              <option value="one-bed">One Bed</option>
```

Comment:
The options value will be the text in between the ></ brackets if you don't specify a value. It may be beneficial to use values that align with your rooms data if you are going to specify them in the HTML.
Something like this:
<option value='1'>One Bed</option>